tests/functional/rbac/actions/test_run_actions.py

```diff
@@ -0,0 +1,247 @@
+from tests.functional.rbac.conftest import BusinessRole, delete_policy, is_denied, is_allowed, as_user_objects
+from tests.functional.rbac.actions.utils import action_business_role, create_action_policy
+
+
+DO_NOTHING_ACTION = 'Do nothing'
+
+CONFIG_FIELD_TO_CHANGE = 'longstring'
+ACTION_CONFIG_ARGUMENT = 'valofarg'
+CHANGE_ACTION_NAME_TEMPLATE = 'Change {object_type} Configuration'
+
+
+def _do_nothing_action_not_presented(obj):
+    """Filter for removing objects that doesn't have "Do nothing" action"""
+    return DO_NOTHING_ACTION not in (a.display_name for a in obj.action_list())
+
+
+# !===== Tests ======!
+
+# pylint: disable-next=too-many-locals
+def test_cluster_basic(sdk_client_fs, user, user_sdk, actions_cluster, simple_cluster):
+    """
+    Test basic permissions on cluster objects' actions:
+      - granted permission allows running only specified action on specified object;
+      - other objects' actions are forbidden to run;
+      - deleting policy actually removes right to run action.
+    """
+    cluster = actions_cluster
+    service = cluster.service(name='actions_service')
+    component = service.component(name='simple_component')
+    all_objects = *get_all_cluster_tree_plain(actions_cluster), *get_all_cluster_tree_plain(simple_cluster)
+
+    for adcm_object in (cluster, service, component):
+        _test_basic_action_run_permissions(adcm_object, sdk_client_fs, user_sdk, user, all_objects)
+
+
+# pylint: disable-next=too-many-locals
+def test_provider_basic(sdk_client_fs, user, user_sdk, actions_provider, simple_provider):
+    """
+    Test basic permissions on provider objects' actions:
+      - granted permission allows running only specified action on specified object;
+      - other objects' actions are forbidden to run;
+      - deleting policy actually removes right to run action.
+    """
+    provider = actions_provider
+    host = actions_provider.host()
+    all_objects = provider, host, simple_provider, simple_provider.host()
+
+    for adcm_object in (provider, host):
+        _test_basic_action_run_permissions(adcm_object, sdk_client_fs, user_sdk, user, all_objects)
+
+
+def _test_basic_action_run_permissions(adcm_object, admin_sdk, user_sdk, user, all_objects):
+    """Check that granting and withdrawn of permission to run action works as expected"""
+    with allure.step(
+        f'Check that granting policy allows to run action "{DO_NOTHING_ACTION}" on {get_object_represent(adcm_object)}'
+    ):
+        (user_object,) = as_user_objects(user_sdk, adcm_object)
+        business_role = action_business_role(adcm_object, DO_NOTHING_ACTION)
+        policy = create_action_policy(admin_sdk, adcm_object, business_role, user=user)
+
+        check_single_action_is_allowed_on_object(DO_NOTHING_ACTION, adcm_object, user_sdk, business_role)
+
+    with allure.step(f"Check that granted permission doesn't allow running '{DO_NOTHING_ACTION}' on other objects"):
+        with user_objects_map(
```

Comment:
maybe check actions explicitly without too complex `user_objects_map` func?

Reply:
At first, I was thinking about `user_objects_map` as a function to be reused to filter different kinds of "denied" objects, but I'm still not sure that it'll be somehow different without this func. I may unwrap it inside the `check_action_is_not_allowed_on_objects`, but do you think it's worth it?